Bio/Align/__init__.py

```diff
@@ -904,13 +1093,13 @@ def substitutions(self):
 
         Any weights associated with the sequences are taken into account when
         calculating the substitution matrix.  For example, given the following
-        multiple sequence alignment::
+        multiple sequence alignment:
```

Comment:
Please revert this change, the double colon is RST markup indicating the following indented code is a literal block. Same for the change below.